tmt/result.py

```diff
@@ -109,26 +120,20 @@ def __init__(
                 self.ids[key] = test.node.get(key)
             interpret = ResultInterpret(test.result) if test.result else ResultInterpret.RESPECT
         else:
-            try:
-                self.ids = data['ids']
-            except KeyError:
-                self.ids = {}
+            self.ids = data.ids or {}
```

Comment:
`or {}` is not necessary, `data` is expected to contain a dictionary, even if it'd be an empty one.